### PR TITLE
changed database to postgres

### PR DESCRIPTION
**Pull request by GangadharSwami**
1. changed database from sqlite to postgres
2. added zip file upload functionality

### CODE REVIEW

File: app/models/client.rb [hunk 1]
```diff
@@ -3,8 +3,15 @@ class Client < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, request_keys: [:subdomain]
+ 
+  validates :subdomain,  presence: true, uniqueness: true
+  validates :email, presence: true, uniqueness: true
+
```
Comment (kapilbhosale) [inline app/models/client.rb]: remove this blank line, not needed.

File: zip_data/Test_Option.csv [hunk 2]
```diff
@@ -0,0 +1,58 @@
+Test_Option_ID,Test_ID,Question_Number,Opt,Date_Of_Creation,Is_Delete
```
Comment (kapilbhosale) [inline zip_data/Test_Option.csv]: Why these csv files are needed in code ?

File: zip_data/Batch_Master.csv [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Batch_ID,Batch_Name,Date_Of_Creation,Date_Of_Modification,Is_Delete
+1,Super Morning- 19 PCB,26/12/2017 12:10:10 PM,31/12/2017 9:50:34 AM,False
```
Comment (kapilbhosale) [inline zip_data/Batch_Master.csv]: Why these files are needed in code?

File: app/models/client.rb (hunk 1 above)
Comment (GangadharSwami) [reply app/models/client.rb]: OK

File: zip_data/Test_Option.csv (hunk 2 above)
Comment (GangadharSwami) [reply zip_data/Test_Option.csv]: These files are generated when we upload  a zip file. I have removed the files as there is no need of these files.

**Pull request merged**